src/persist/setup_test_env_mz.sh

```diff
@@ -15,3 +15,7 @@
 export MZ_PERSIST_EXTERNAL_STORAGE_TEST_S3_BUCKET="mtlz-test-persist-1d-lifecycle-delete"
 export AWS_DEFAULT_REGION="us-east-2"
 export AWS_PROFILE="mz-scratch-admin"
+
+if [[ ! -z "${MZDEV_POSTGRES}" ]]; then
```

Comment:
dunno if you saw this before, but... "booleans are hard!"

Reply:
This should work in Bash?

```bash
if [[ "$MZDEV_POSTGRES" ]]; then
  # ...
fi
```